tensorrec/tensorrec.py

```diff
@@ -387,7 +388,8 @@ def _build_tf_graph(self, n_user_features, n_item_features):
         # Get node names for each graph hook
         for graph_tensor_hook_attr_name in self.graph_tensor_hook_attr_names:
             hook = self.__getattribute__(graph_tensor_hook_attr_name)
-            self.graph_tensor_hook_node_names[graph_tensor_hook_attr_name] = hook.name
+            if hook is not None:
+                self.graph_tensor_hook_node_names[graph_tensor_hook_attr_name] = hook.name
```

Comment:
This check is to skip the things that I added that only apply if it's `biased`, but I was thinking it would be better to have a separate list of `biased_graph_tensor_hook_attr_names` or something like that, which is only used `if self.biased`. Thoughts?

Reply:
I ended up doing this in 2a08a34.